package.json

```diff
@@ -7,6 +7,10 @@
   "main": "./lib/index.js",
   "types": "dist/jiti.d.ts",
   "bin": "bin/jiti.js",
+  "exports": {
+    "./": "./",
+    "./register": "./lib/register.js"
```

Comment:
Last thing since still exports is not available everywhere, what if we move `register.js` to top level to make it working out of the box with `require('jiti/register')` everywhere?

Reply:
Good idea 👍